server/src/main/scala/slickdb/Tables.scala

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Another accidental empty line here

Reply:
This was suggestion made by scala style . Usually first line of any class is some description /** */  . I  kept blank line with which scalastyle was happy with .